src/pages/TCY/components/Stake/Stake.tsx

```diff
@@ -73,7 +73,7 @@ export const Stake: React.FC<TCYRouteProps & { activeAccountNumber: number }> =
 
   return (
     <FormProvider {...methods}>
-      <MemoryRouter initialEntries={StakeEntries} initialIndex={0}>
+      <MemoryRouter initialEntries={StakeEntries} initialIndex={1}>
```

Comment:
@reallybeard not sure we want this?
<img width="1410" alt="image" src="https://github.com/user-attachments/assets/d0735fff-343a-4036-a468-b83545b686a3" />